packages/core/src/create/release.ts

```diff
@@ -148,7 +148,7 @@ export const createReleaseJson = async (
       "2": strings.new_in_version.replace(/^\s+|\s+$/g, ''),
       "3": strings.saga_features?.replace(/^\s+|\s+$/g, ''), // saga_features is optional
       "4": strings.name.replace(/^\s+|\s+$/g, ''),
-      "5": strings.short_description.replace(/^\s+|\s+$/g, ''),
+      "5": strings.short_description?.replace(/^\s+|\s+$/g, ''),
```

Comment:
If this is optional, can we add a comment afterwards to that effect (like on line 149)? The `?` can be easy to miss when reading this code. Also, if this can be undefined, doesn't that also need to be handled at line 133, lest we suffer the same bug as line 131?

Reply:
Hmm, actually, `short_description` is required in releaseJsonMetadata.json; under what circumstances would this be undefined?

Reply:
Ugh, I need to kill this PR. There's a different issue.